### PR TITLE
feat: Custom attributes / properties

### PR DESCRIPTION
**Pull request by kof**
Closes https://github.com/webstudio-is/webstudio/issues/1012

## Description

I decided to just use the same control to add a custom property/attribute, it works same way as adding style sources.

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/builder/features/settings-panel/controls/text.tsx [hunk 1]
```diff
@@ -44,9 +47,16 @@ export const TextControl = ({
     prop?.type === "expression" ? prop.value : undefined
   );
 
+  useEffect(() => {
+    if (autoFocus && textAreaRef.current) {
+      textAreaRef.current.focus();
```
Comment (TrySound) [inline apps/builder/app/builder/features/settings-panel/controls/text.tsx:52]: textAreaRef.current?.focus()

**Comment by TrySound** [comment]
I would align combobox list to left side of input
image

**Comment by TrySound** [comment]
Cannot bind expression
image

**Comment by kof** [comment]
> I would align combobox list to left side of input

It was like that before, but I agree

**Review by TrySound — APPROVED** [review]

**Pull request merged**